packages/eslint-plugin/lib/utilities/index.js

```diff
@@ -147,6 +147,7 @@ module.exports = {
   polarisComponentFromJSX,
   getRootObject,
   docsUrl,
+  findDefinition,
```

Comment:
Where is this used, can it be removed? It does not seem to be a top-level function in this file so iI'd be suprised if this even works.